test/post-911-gib-status/01-authed.e2e.spec.js

```diff
@@ -0,0 +1,40 @@
+const E2eHelpers = require('../e2e/e2e-helpers');
+const Timeouts = require('../e2e/timeouts.js');
+const GibsHelpers = require('../e2e/post-911-gib-status-helpers.js');
+const LoginHelpers = require('../e2e/login-helpers');
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    const token = LoginHelpers.getUserToken();
+
+    GibsHelpers.initApplicationMock(token);
+
+    // Ensure main status page renders.
+    LoginHelpers.logIn(token, client, '/education/gi-bill/post-9-11/ch-33-benefit', 3)
+      .waitForElementVisible('body', Timeouts.normal)
+      .axeCheck('.main')
+      .assert.title('Check Benefit: Vets.gov')
+      .waitForElementVisible('.schemaform-title', Timeouts.slow);  // First render of React may be slow.
+
+    // Print page
+    client
+      .click('#print-button')
+      .windowHandles((result) => {
+        const newWindow = result.value[1];
+        client.switchWindow(newWindow);
+      });
+
+    // Ensure main status page renders.
```

Comment:
Change comment to "Ensure print page renders"

We probably want a comment explaining that Electron loses the session data so in the test environment we need to log back in to see the print page.